# ┌─────────────────────────────────────────────────────────────────────────────────────
# │ PROJECT IMPORTS
# └─────────────────────────────────────────────────────────────────────────────────────

from core.exceptions import Error


# ┌─────────────────────────────────────────────────────────────────────────────────────
# │ DOES NOT EXIST ERROR
# └─────────────────────────────────────────────────────────────────────────────────────


class DoesNotExistError(Error):
    """Raised when an expected resource does not exist"""


# ┌─────────────────────────────────────────────────────────────────────────────────────
# │ DUPLICATE KEY ERROR
# └─────────────────────────────────────────────────────────────────────────────────────


class DuplicateKeyError(Error):
    """Raised when a resource with a duplicate key is found"""
